qutebrowser/components/adblock.py

```diff
@@ -206,53 +238,26 @@ def _import_local(self, filename: str) -> None:
         self._in_progress.append(download)
         self._on_download_finished(download)
 
-    def _parse_line(self, raw_line: bytes) -> bool:
+    def _parse_line(self, raw_line: bytes) -> list:
```

Comment:
I don't think this needs to return a value now, right? That value isn't used anywhere anyways.

Reply:
Alternatively it's possible to return it here and add it to blocked host list outside of the function.

Reply:
See my comment below - I don't think this function will is needed at all after refactoring things.